### ISSUE TITLE
Move Accessibility Score Calculations Out of ApiController.scala

### ISSUE THREAD

**Issue by jonfroehlich** [ProjectSidewalk/SidewalkWebpage#3868]
I do not think that the actual calculations should be done in ApiController.scala as they are now (indeed, we have computeAccessScoresForNeighborhoods, computeAccessScoresForStreets, and computeAccessScore as methods within ApiController.scala but that's not what that class is for!). Instead, we should have an independent module that calculates accessibility scores that the ApiController then uses.

Related to but not the same as: https://github.com/ProjectSidewalk/SidewalkWebpage/issues/3855

**Comment by misaugstad** [comment]
Closing via #3894
